src/Bab/RabbitMq/Specification/DeadLetterExchangeCanBeCreated.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+namespace Bab\RabbitMq\Specification;
+
+class DeadLetterExchangeCanBeCreated implements Specification
+{
+    public function isSatisfiedBy($config)
+    {
+        if ($config->hasDeadLetterExchange() === true) {
+            return true;
+        }
+
+        if (!isset($config['queues']) || 0 === count($config['queues'])) {
+            return false;
+        }
+
+
+        $currentWithDl = false;
```

Comment:
not needed.

if isset(parameters['retries']) => return true
if isset(parameters['with_dl'] && true === with_dl) => return true